README.md

```diff
@@ -71,7 +71,52 @@ The current version for each package can be found in the respective `package.jso
 
 ### Application Owner
 
-To get started tracing your own application, see the [Getting Started Guide](getting-started/README.md). For more information about automatic instrumentation see [@opentelemetry/node][otel-node], which provides auto-instrumentation for Node.js applications. If the automatic instrumentation does not suit your needs, or you would like to create manual traces, see [@opentelemetry/tracing][otel-tracing]
+#### Install Dependencies
+
+```shell
+npm install --save @opentelemetry/api
+npm install --save @opentelemetry/node
+npm install --save @opentelemetry/tracing
+npm install --save @opentelemetry/auto-instrumentations-node
+```
+
+#### Instantiate Tracing
+
+```js
+// tracing.js
+
+'use strict'
+
+const { getNodeAutoInstrumentations } = require("@opentelemetry/auto-instrumentations-node");
+const { registerInstrumentations } = require('@opentelemetry/instrumentation')
+const { ConsoleSpanExporter, SimpleSpanProcessor } = require('@opentelemetry/tracing')
+const { NodeTracerProvider } = require('@opentelemetry/node')
+const { Resource } = require('@opentelemetry/resources');
+const { ResourceAttributes } = require('@opentelemetry/semantic-conventions');
```

Comment:
```suggestion
const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
```

Reply:
Updated in new commit (assuming #2345 will release ahead of this)